civicaseextras.civix.php

```diff
@@ -485,3 +485,13 @@ function _civicaseextras_civix_civicrm_entityTypes(&$entityTypes) {
   $entityTypes = array_merge($entityTypes, array (
   ));
 }
+
+function _civicaseextras_civicrm_alterAngular(\Civi\Angular\Manager &$angular) {
+  $changeSet = \Civi\Angular\ChangeSet::create('inject_case_outcomes')
+    ->alterHtml('~/civicase/CaseDetails--tabs--summary--CustomData.html',
+      function (phpQueryObject $doc) {
+        $doc->find('[civicase-masonry-grid]')
+          ->prepend('<civicase-extras-case-outcome case="item"></civicase-extras-case-outcome>');
```

Comment:
what's this all for?

Reply:
Sorry, I should have added this to the description.

This is how an extension can modify the AngularJS templates of another extension. In this case the the case outcomes panel is being added to the grid in the custom data partial.